### PR TITLE
Replace gulp-webserver with gulp-connect

### PR DESCRIPTION
**Pull request by naknode**
This adheres issue #195 and successfully replaces said NPM package (which had 2 bad vulns). When I ran `npm audit`, it had no more security concerns.

```bash
➜  ~/code/oss/desktopJS git:(master) npm audit

                       === npm audit security report ===

found 0 vulnerabilities
 in 45568 scanned packages
```

---

Where can I find the Contributor License Agreement? Would I just talk to the email (desktopjs@morganstanely.com) asking for it?

### CODE REVIEW

**Comment by bingenito** [comment]
Thanks @naknode , I'll reply back on your mail about the CLA.

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ To run the examples or to manually test scenarios in each container, there is a
 a local server with live reload.  This can be run directly via gulp or through npm.
 
 ```
-$ npm run server
+$ npm run start
```
Comment (bingenito) [inline README.md]: Excellent.  I changed/broke that on the monorepo refactor

**Comment by naknode** [comment]
> Thanks @naknode , I'll reply back on your mail about the CLA.

Great! Can't wait to start contributing to Morgan Stanley and this project!

**Comment by naknode** [comment]
DCO has been added and signed.

**Comment by bingenito** [comment]
@naknode As it is required to indicate on each commit that it is covered by <dco>, I'll need updated commit messages.  You can squash down to one commit if that is easier.

File: gulpfile.js [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 gulp.task('test', require('./.gulp/tasks/tests')(gulp, gulpConfig));
 
 gulp.task("server", function () {
+	connect.server({
+		root: 'examples/web/',
+		port: 8000,
+		livereload: true
+  	});
+
     return gulp.src(['examples/web', 'packages/desktopjs/dist', 'packages/desktopjs-electron/dist', 'packages/desktopjs-openfin/dist'])
```
Comment (bingenito) [inline gulpfile.js]: @naknode These other paths don't seem to be served so I get 404 errors in the examples.

Comment (bingenito) [reply gulpfile.js]: Try moving the array to be the value for root and remove the return

Comment (naknode) [reply gulpfile.js]: That seems to have worked as I can successfully navigate around the application.

In my terminal:

```bash
[11:28:44] Using gulpfile ~/code/oss/desktopJS/gulpfile.js
[11:28:44] Starting 'server'...
[11:28:44] Starting server...
[11:28:44] Server started http://localhost:8000
[11:28:44] LiveReload started on port 35729
[11:28:44] Running server
```

Please let me know what command you are using to generate those errors. I do realize that when I do `npm run build`, I get `TypeScript: 503 semantic errors`

Comment (bingenito) [reply gulpfile.js]: I just went opened localhost:8000 and nothing worked so I looked at dev tools and saw all the 404

Comment (naknode) [reply gulpfile.js]: Even now? I just did a fresh install of my @naknode/desktopJS and I was able to get localhost:8000 up and running with gulp-connect.

Comment (bingenito) [reply gulpfile.js]: It's all good, sorry for confusion.  I thought you were asking how I knew it wasn't working.  Only thing left is to update each commit to indicate dco per contributing guide (I suggest squashing to make that easier).

**Comment by naknode** [comment]
Would you mind showing me how to squash those commits? I'm trying to rebase and I thought I could do, but git rebase is proving to be more tricky than I thought. https://github.com/Morgan-Stanley/desktopJS/pull/220#issuecomment-462463523 is where I failed, too. Thank you.

**Comment by bingenito** [comment]
@naknode Does this help - https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git ?

**Comment by naknode** [comment]
@bingenito Believe me, I tried. And tried. Git is a fickle thing. Maybe you can see #220 and see if that is applicable to be merged?

**Comment by bingenito** [comment]
I need the dco on your side on each commit message to cover the submission.  I think my merge of my PR in the middle is making this harder.  When all else fails I usually just create a new PR.

**Comment by naknode** [comment]
Closing.

**Pull request closed without merging**